src/classes/PS_IntegrationService_TEST.cls

```diff
@@ -35,16 +35,112 @@
 * @description Test class for the PS_IntegrationService class
 */
 
-@isTest
+@IsTest
 private class PS_IntegrationService_TEST {
-    
+
+    private static Map<String, List<Map<String, String>>> configParams;
+    private static Map<String, PS_IntegrationService.Config> makanaConfigParams;
+    private static Map<String, PS_IntegrationService.Config> paymentConfigParams;
+    private static final String API_KEY = 'apiKey';
+    private static final String PAYMENTS_APPLICATION_TYPE = 'payments';
```

Comment:
@voduyemi everything is looking pretty good here, but did we ever catch up on why we are storing separate config for Makana and Payments? Or is it just that we are set up to support multiple configurations for possible future applications using this object, and we are using these two as test examples?

Reply:
After speaking to Pedro, he confirmed that the transport layer will always send  both payment and makana configurations, the makana configuration is meant to hold a fallback api key that can be used if we ever receive payment configuration without an api key.